src/server/game/Chat/Channels/Channel.h

```diff
@@ -269,19 +284,21 @@ class TC_GAME_API Channel
         }
 
         typedef std::map<ObjectGuid, PlayerInfo> PlayerContainer;
-        typedef GuidSet BannedContainer;
-
-        bool _announce;
-        bool _ownership;
-        bool _IsSaved;
-        uint8 _flags;
-        uint32 _channelId;
-        uint32 _Team;
-        ObjectGuid _ownerGUID;
-        std::string _name;
-        std::string _password;
-        PlayerContainer playersStore;
-        BannedContainer bannedStore;
+        typedef GuidUnorderedSet BannedContainer;
+
+        bool AnnounceEnabled;           //< Whether we should broadcast a packet whenever a player joins/exits the channel
```

Comment:
all of these names should start with underscore and lowercase letter if they are not public